commands/copilot_cli.py

```diff
@@ -66,6 +66,7 @@ def _normalise_keys(source: dict):
             if valid_services != normalised_config[storage_name]["services"]:
                 normalised_config[storage_name]["services"] = valid_services
                 click.echo(click.style(f"Services listed in {storage_name} do not exist in ./copilot/", fg="red"))
+                exit()
```

Comment:
I missed this error because there was a ton of output including the instructions after it.

Better to exit with the error in your face.

Reply:
100% agree with this. Errors should be mitigated at the earliest opportunity.

Reply:
Add non zero exit code...
```suggestion
                exit(1)
```